tuxemon/states/techniques/__init__.py

```diff
@@ -25,7 +24,11 @@ class TechniqueMenuState(Menu[Technique]):
     background_filename = "gfx/ui/item/item_menu_bg.png"
     draw_borders = False
 
-    def __init__(self) -> None:
+    def __init__(self, **kwargs) -> None:
```

Comment:
Don't really need to pass it via kwargs. Just do:

`def __init__(self, monster: Optional[Monster] =None) -> None:`

Reply:
Ok